solana/ts/sdk/ntt.ts

```diff
@@ -640,17 +641,30 @@ export class SolanaNtt<N extends Network, C extends SolanaChains>
 
     const payer = new SolanaAddress(sender).unwrap();
 
-    const ix = await NTT.createInitializeInstruction(
-      this.program,
-      {
-        ...args,
-        payer,
-        owner: payer,
-        chain: this.chain,
-        tokenProgram: mintInfo.owner,
-      },
-      this.pdas
-    );
+    const ix = args.multisig
+      ? await NTT.createInitializeMultisigInstruction(
+          this.program,
+          {
+            ...args,
+            payer,
+            owner: payer,
+            chain: this.chain,
+            tokenProgram: mintInfo.owner,
+            multisig: args.multisig,
+          },
+          this.pdas
+        )
+      : await NTT.createInitializeInstruction(
+          this.program,
+          {
+            ...args,
+            payer,
+            owner: payer,
+            chain: this.chain,
+            tokenProgram: mintInfo.owner,
+          },
+          this.pdas
+        );
```

Comment:
```suggestion
const createInstruction = args.multisig
  ? NTT.createInitializeMultisigInstruction
  : NTT.createInitializeInstruction;

const ix = await createInstruction(
  this.program,
  {
    ...args,
    payer,
    owner: payer,
    chain: this.chain,
    tokenProgram: mintInfo.owner,
    ...(args.multisig && { multisig: args.multisig }),
  },
  this.pdas
);
```
Making this DRYer - not sure if the suggestion actually works, but it gets the idea across.

Reply:
This is definitely easier to follow but TS screams at me with an error when I tried this:
```
...
Types of property 'multisig' are incompatible.
   Type 'PublicKey | undefined' is not assignable to type 'PublicKey'.
      Type 'undefined' is not assignable to type 'PublicKey'.
```
I tried passing the `multisig` as not part of `args` object but even then it had the same error.

Reply:
The issue seems to be that the condition deciding`createInstruction` (i.e., `args.multisig`) is not "propagated" when `createInstruction` is used. 
So despite using the same condition (without any changes to `args.multisig` in between), TS is not able to figure out which version of `createInstruction` is used (it assumes both could be used) and thus gives me the error.

Reply:
Yeah, I figured that there might be a TS issue there (hence my uncertainty) but I'd assume there's still a way to do away with a bunch of repetition. Overall it's not a big deal ofc.